aio_pika/robust_connection.py

```diff
@@ -44,7 +44,13 @@ def __init__(self, url, loop=None, **kwargs):
 
         self.__channels = set()
         self._reconnect_callbacks = CallbackCollection()
+        self._connect_lock = asyncio.Lock()
         self._closed = False
+        self.connected = asyncio.Event()
```

Comment:
What this event for?

Reply:
Event for external use. Useful for tests right now but I want to document it.